_data/backgrounds.yml

```diff
@@ -950,3 +950,137 @@
     slurpodromeStart: mp9_SRP_START
     slurpodromeRace: mp9_SRP_RACE
     slurpodromeWin: mp9_SRP_WIN
+- background: mkwii_funkystadium
+  name: Funky Stadium
+  download:
+  - https://nikkums.io/cswt/Backgrounds/mkwii_funkystadium.background.zip
+  - https://drive.google.com/uc?export=download&id=17tEyDEDZPSICRHv940ejO7vswqWvx-5R&confirm=1
```

Comment:
Since all of these tracks live within the background .zips for each map, you can actually omit the `download` key here since this music will be downloaded alongside the background. You'll still need to specify when each song should play, but no extra download is needed since you're just using the background's music directly.